dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/tckRunner/tck/standalone.pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Copyright (c) 2022 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<!-- NOTE: This pom is used for Standalone Testing -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.jakarta.enterprise.concurrent</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>Jakarta Concurrency TCK Runner TCK Module</name>
+
+    <properties>
+        <jakarta.concurrent.version>3.0.0</jakarta.concurrent.version>
+        
+        <arquillian.version>1.6.0.Final</arquillian.version>
+        <arquillian-wlp.version>2.0.2</arquillian-wlp.version>
+        
+        <testng.version>6.14.3</testng.version>
+
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <!--  the below is used in arquillian.xml -->
+        <wlp>${wlp}</wlp>
+        
+        <targetDirectory>${project.basedir}/target</targetDirectory>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>${arquillian.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <!-- TCK Dependency -->
+        <dependency>
+            <!-- TODO: Update this once TCK is available on maven -->
```

Comment:
In the meantime, use the copy of it that we uploaded to DHE:
```
		    <groupId>io.openliberty.jakarta.enterprise.concurrent</groupId>
		    <artifactId>jakarta.enterprise.concurrent-tck</artifactId>
		    <version>3.0.0.20220120</version>
```

The presence of the DHE version number is how we will figure out all of the places to update once everything is on Maven.